streamer/Cargo.toml

```diff
@@ -7,7 +7,7 @@ authors = { workspace = true }
 repository = { workspace = true }
 homepage = { workspace = true }
 license = { workspace = true }
-edition = { workspace = true }
+edition = "2024"
```

Comment:
Hi @puhtaytow - As mentioned in [this comment](https://github.com/anza-xyz/agave/issues/6203#issuecomment-3051035242) and corroborated by CI failing in this PR, we are blocked on this.

I appreciate the enthusiasm but please hold off on making any more 2024-edition bump PR's at the moment. I will comment in https://github.com/anza-xyz/agave/issues/6203 when we're good to proceed if you're still interested in working on these